deployment/helm/templates/deployment.yaml

```diff
@@ -81,6 +81,10 @@ spec:
             value: "/secrets/github-app/github_app_webhook_secret"
           - name: "MINDER_PROVIDER_GITHUB_APP_FALLBACK_TOKEN_FILE"
             value: "/secrets/github-app/github_app_fallback_token"
+          - name: "MINDER_WEBHOOK_CONFIG_WEBHOOK_SECRET_FILE"
+            value: "/secrets/app/repo_webhook_secret"
```

Comment:
```suggestion
            value: "/secrets/app/github_repo_webhook_secret"
```

Reply:
🤦🏻 of course I copied the AWS secret names and not the k8s secret names.
Thank you!